src/main.rs

```diff
@@ -96,20 +124,47 @@ async fn main() {
             tracing::info!(pid = std::process::id());
             poller::start().await;
         }
-        Some(("watch", _)) => {
-            watch_dir(&dir);
+        Some(("watch", m)) => {
```

Comment:
nit: This scope is too big for a single-letter variable. My brain can't wrap around what `m` is supposed to mean.